api/construction_service.go

```diff
@@ -435,6 +455,15 @@ func (s *Server) ConstructionPreprocess(ctx context.Context, r *types.Constructi
 	if xerr != nil {
 		return nil, xerr
 	}
+	// NOTE(tav): We explicitly error on transfers to the fee address so as to
```

Comment:
Could the staking require to transfer fund to fee addr? We can worry about this later though.

Reply:
Yeah, sorry, I'm ignoring staking for now.